app/views/users/_user_organisations_list.html.erb

```diff
@@ -1,17 +1,44 @@
+
 <div id="organisations_list" class="col-12 col-md-6 px-5">
   <h4>
     <%= User.human_attribute_name(:organisations) %>
   </h4>
-  <% user.organisations.each do |organisation|  %>
-    <p class="mb-2">
-      <%= organisation %>
-      <% if @department.id != organisation.department_id %>
-        <small>
-          (Organisation d'un autre départment : <%= organisation.department.number %> - <%= organisation.department.name %>)
-        </small>
+  
+  <p class="mb-3 mt-3 flex-wrap d-flex" id="organisations_list">
+    <% if user.organisations.blank? %>
+      -
+    <% else %>
+      <% user.organisations.each do |organisation| %>
+        <span class="badge badge-tag justify-content-between background-blue-light text-dark-blue me-2 d-flex">
+          <%= organisation.name %>
+          <% if @department != organisation.department %>
+            <small>
+              (Organisation d'un autre départment : <%= organisation.department.number %> - <%= organisation.department.name %>)
+            </small>
+          <% end %>
+          <%
+            if user.organisations.count > 1
+              data = {
+                turbo_confirm: "Retirer l'usager de cette organisation ?",
+                turbo_confirm_text_content: "Êtes-vous sûr de vouloir retirer l'usager <b>#{user}</b> de l'organisation <b>#{organisation.name}</b> ?",
+                turbo_confirm_text_action: "Retirer",
+                turbo_method: :delete,
+              }
+            else
+              data = {
+                turbo_confirm: true,
+                turbo_confirm_template: raw(render("users/user_remove_from_last_organisation_confirm", user: user, organisation: organisation)),
```

Comment:
Ah oui tu mets carrément l'html en data attribute :D, why not l'api est cool en tous cas 👍